.circleci/workflows.yml

```diff
@@ -1167,6 +1181,12 @@ workflows:
             branches:
               only:
                 - main
+      - artifact-deployment:
+          name: Trigger bqetl_artifact_deployment Airflow DAG
+          context:
+            - data-eng-circleci-deployment
+          requires:
+            - deploy
```

Comment:
For consistency/clarity I'd suggest adding a `main` branch filter (e.g. the `deploy` job has such a filter even though it requires the `deploy-to-private-gcr` job which already has such a filter).